lib/screens/app/app.dart

```diff
@@ -114,55 +127,137 @@ class _AppState extends State<App> with WidgetsBindingObserver {
       case AppLifecycleState.paused:
         break;
       case AppLifecycleState.resumed:
-        Provider.of<ConnectionNotifier>(context, listen: false).discoverEndpoints();
+        Provider.of<ConnectionNotifier>(context, listen: false)
+            .discoverEndpoints();
         break;
       case AppLifecycleState.detached:
         break;
     }
   }
 
+  void processIdentityRequest(request) async {
+    if (request.manager.data.req[1] == null ||
```

Comment:
No idea what this logic checks. 
Move the logic into a variable with a meaningful name before using it.